tests/validators/literal.test.ts

```diff
@@ -13,10 +13,20 @@ describe('LiteralValidator', () => {
 		);
 	});
 
-	test('Clone', () => {
+	test('GIVEN clone THEN returns similar instance', () => {
 		// @ts-expect-error Test clone
 		const clonePredicate = predicate.clone();
 
-		expect(clonePredicate.parse('sapphire')).toBe('sapphire');
+		expect(clonePredicate).toBeInstanceOf(predicate.constructor);
+	});
+
+	describe('DateLiteral', () => {
+		const date = new Date('2022-01-01');
+		const dateLiteralPredicate = s.literal(date);
+		const datePredicate = s.date;
+
+		test('GIVEN a date literal THEN returns the given value', () => {
+			expect(dateLiteralPredicate.parse(date)).toStrictEqual(datePredicate.parse(date));
```

Comment:
Should test for instances with the same value, not just the instance used in the literal.

Reply:
you mean `toStrictEqual(date)`?

Reply:
As a matter of a fact, I highly doubt Jest checks date equality properly, you may need `.getTime()`.

But no, I'm referring that you should construct a new instance with the exact same time, and see that it matches.